pkg/traceql/engine_test.go

```diff
@@ -386,3 +386,80 @@ func TestExamplesInEngine(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractMatchers(t *testing.T) {
+	testCases := []struct {
```

Comment:
test multiple spansets and pipelined queries

Reply:
Added!